Frameworks/UIKit/UIPresentationController.mm

```diff
@@ -19,24 +19,51 @@
 
 #include "UIKit/UIPresentationController.h"
 
-@implementation UIPresentationController
+@implementation UIPresentationController {
+    StrongId<UIViewController> _presentingViewController;
+    StrongId<UIViewController> _presentedViewController;
+    StrongId<UIView> _containerView;
+}
 
 /**
  @Status Interoperable
 */
 - (instancetype)initWithPresentedViewController:(UIViewController*)presentedViewController
              presentingViewController:(UIViewController*)presentingViewController {
-    _presentingViewController = presentingViewController;
-    _presentedViewController = presentedViewController;
-    _presentationStyle = _presentedViewController.modalPresentationStyle;
-    _containerView = _presentingViewController.view;
+    if (self = [super init]) {
+        _presentingViewController = presentingViewController;
```

Comment:
I guess this is not a problem since both objects exist on UIViewControllerInternal for the lifetime of the the UIViewControllerPriv.

---
In reply to: [89899882](https://github.com/Microsoft/WinObjC/pull/1464#discussion_r89899882) [](ancestors = 89899882)